<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detuTittleWgt</class>
 <widget class="QWidget" name="detuTittleWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>43</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>36</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>43</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background:#323232;
box-shadow:inset 0 1px 0 0 rgba(167,167,170,0.20);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>19</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="logo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>21</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#logo{
border-image:url(:/image/image/monitor.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="logoLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#logoLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#eeeeee;
line-height:12px;
text-align:left;
}</string>
           </property>
           <property name="text">
            <string>Pano-VMS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="styleSheet">
         <string notr="true">#timeLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#eeeeee;
text-align:left;
}</string>
        </property>
        <property name="text">
         <string>14:28:30</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="unlockBtn">
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="toolTip">
           <string>锁定</string>
          </property>
          <property name="styleSheet">
           <string notr="true">#unlockBtn{
border-image: url(:/image/image/unlock.png);
}
#unlockBtn:pressed{
margin:1px;
}
#unlockBtn:hover{
border-image:url(:/image/image/unlockb.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingBtn">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
          <property name="toolTip">
           <string>系统设置</string>
          </property>
          <property name="styleSheet">
           <string notr="true">#settingBtn{
border-image: url(:/image/image/setting.png);
}
#settingBtn:hover{
border-image: url(:/image/image/settingb.png);
}
#settingBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpBtn">
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="toolTip">
           <string>帮助</string>
          </property>
          <property name="styleSheet">
           <string notr="true">#helpBtn{
border-image: url(:/image/image/help.png);
}
#helpBtn:pressed{
margin:1px;
}
#helpBtn:hover{
border-image: url(:/image/image/helpb.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="sigWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#sigWidget{
opacity:0.3;
background:#000000;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>18</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="minimizeBtn">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#minimizeBtn{
border-image:url(:/image/image/windowmin.png);
}
#minimizeBtn:pressed{
margin:1px;
}
#minimizeBtn:hover{
border-image:url(:/image/image/windowminb.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="maxmizeBtn">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#maxmizeBtn{
border-image:url(:/image/image/windowbiger.png);
}
#maxmizeBtn:pressed{
margin:1px;
}
#maxmizeBtn:hover{
border-image:url(:/image/image/windowbigerb.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#closeBtn{
border-image:url(:/image/image/windowx.png);
}
#closeBtn:hover{
border-image:url(:/image/image/windowxb.png);
}
#closeBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="detupvms.qrc"/>
 </resources>
 <connections/>
</ui>
